khmer/khmer_args.py

```diff
@@ -110,6 +272,16 @@ def __call__(self, parser, namespace, values, option_string=None):
 ** Your values for ksize, n_tables, and tablesize
 ** will be ignored.'''.format(hashfile=values))
 
+            if getattr(namespace, 'unique_kmers') != 0 or \
```

Comment:
is there a reason not to use 'namespace.unique_kmers' and 'namespace.max_memory_usage' here?